bedrock/firefox/views.py

```diff
@@ -313,6 +313,23 @@ def show_57_dev_whatsnew(version):
     return version >= Version('57.0')
 
 
+def show_57_whatsnew(version, oldversion):
+    try:
+        version = Version(version)
+        if oldversion:
+            oldversion = Version(oldversion)
+    except ValueError:
+        return False
+
+    v57 = Version('57.0')
+    v58 = Version('58.0')
+
+    if oldversion:
+        return version >= v57 and version < v58 and oldversion < v57
+    else:
+        return version >= v57
```

Comment:
Should this be `return version == v57`? 

Currently this always shows the cbeard letter e.g. for 57.0.4 and 58.0 etc. Some tests for this logic would also be good?